packages/cozy-sharing/src/components/AvatarPlusX.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { SharingTooltip, TooltipRecipientList } from './Tooltip'
+import Avatar from 'cozy-ui/transpiled/react/Avatar'
+
+const AvatarPlusX = ({ size, extraRecipients = [] }) => (
+  <div data-tip data-for="extra-recipients-avatar">
+    <Avatar
+      size={size}
+      text={`+${Math.min(extraRecipients.length, 99)}`}
+      style={{ backgroundColor: 'var(--silver)', color: 'var(--charcoalGrey)' }}
```

Comment:
we should not use --silver / --charcoalGrey, can't we use primary and so on?

Reply:
We don't have semantic color names for this use case (it's for example unrelated to the primaryTextColor) — imo it's a combination specific to these cozy-sharing recipients. 

What do you think of adding semantic color names for this in cozy-sharing (eg. `--genericRecipientBackground` and `--genericRecipientText`)? That way the names are semantic and we can override them with a theme.

Reply:
I thinks it's better yeah.